src/client/src/rt-util/index.ts

```diff
@@ -1,5 +1,6 @@
 import { ActionCreatorsMapObject } from 'redux'
 export { withDefaultProps } from './reactTypes'
 import { action, ActionUnion } from './ActionHelper'
+export * from './utilityTypes'
```

Comment:
export * breaks the convention.  The convention is `import {someTypes} from someModule` then `export type someTypes = someTypes`.